### PR TITLE
Form not needed

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10102

source https://forms.office.com/pages/responsepage.aspx?id=v4j5cvGGr0GRqy180BHbR1ax4zKyZjVBmutzKVo1pVtURTVXM0wzSldCTjVERDlYRE1WWktQRDhURCQlQCN0PWcu

### CODE REVIEW

File: Teams/export-teams-content.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Here are some examples on how you can use these export APIs:
 
 ## Prerequisites to access Teams Export APIs
 
-- Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. Export APIs require that you have additional validation, beyond permissions and consent, before you can use them. To request access to these protected APIs, complete the [request form](https://aka.ms/teamsgraph/requestaccess).
+- Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. You can immediately call the protected APIs as long as the requirements for [accessing without a user](/graph/auth-v2-service) are met, the online form and the protected API approval process are no longer needed.
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:74]:
```suggestion
- Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. You can immediately call the protected APIs as long as the requirements for [accessing without a user](/graph/auth-v2-service) are met, the online form and the protected API approval process are no longer needed.
```
```suggestion
- Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. You can immediately call the protected APIs as long as the requirements for [accessing without a user](/graph/auth-v2-service) are met, the online form and the protected API approval process are no longer needed.
```
```suggestion
- Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. You can immediately call the protected APIs as long as the requirements for [accessing without a user](/graph/auth-v2-service) are met. The online form and the protected API approval process are no longer needed.
```

Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:74]:
```suggestion
- Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. You can immediately call the protected APIs as long as the requirements for [accessing without a user](/graph/auth-v2-service) are met. The online form and the protected API approval process are no longer needed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**